src/res/heatmap.html

```diff
@@ -0,0 +1,4289 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='utf-8'>
```

Comment:
Size of the file matters, since it is a part of the profiler package.
I therefore suggest using TABs in HTML to reduct the file size.

Reply:
done